### PR TITLE
feat: add MetaWorld MT50 evaluation and training support

### PR DESCRIPTION
**Pull request by DorayakiLin**
Add MetaWorld MT50 benchmark integration following the simBenchmarks convention. Includes eval scripts (tyro CLI), ModelClient adapter, shell wrappers, training config template, and data pipeline registration (MetaWorldRobotDataConfig + metaworld_mt50 mixture).

## Motivation

Closes #

## Changes

-
-

## Testing

- [ ] Local training for N steps without errors
- [ ] Benchmark evaluation results (**required** for framework / benchmark changes, see table below)

| Benchmark | Metric | Result |
|-----------|--------|--------|
|           |        |        |

- **Checkpoint**: 
- **Config**: 
- **Reproduce**: 

## Type of Change
- [ ] Bug fix (non-breaking)
- [ ] New feature (non-breaking)
- [ ] New framework / benchmark integration
- [ ] Breaking change
- [ ] Documentation only
- [ ] Refactor (no behavior change)

## Checklist

- [ ] No unrelated changes mixed in
- [ ] New features have example config in `examples/`
- [ ] No secrets, API keys, or large binaries committed
- [ ] Files stay isolated — no unnecessary modification to shared modules (`starVLA/dataloader/`, `starVLA/training/`, `starVLA/config/`)
- [ ] No modifications to shared files, or justified above
- [ ] If adding framework/benchmark: checkpoint uploaded to personal HF (public)

### CODE REVIEW

**Comment by DaojunZhu** [comment]
@DorayakiLin 你好,  readme中提到预训练piv3对结果提升很大, 请教下, 预训练是怎么训练的呢, 基于什么数据呢, 另外预训练checkpoint可以提供么

**Comment by DorayakiLin** [comment]
预训练具体信息请看LA4VLA论文（https://arxiv.org/abs/2606.27295），我们使用Language-Action Pretraining方式在LA-33K数据集上对starvla进行预训练。预训练checkpoint我稍后整理一下传到https://huggingface.co/collections/MINT-SJTU/la4vla路径。
